rmw_iceoryx_cpp/src/internal/iceoryx_name_conversion.cpp

```diff
@@ -159,9 +159,9 @@ get_iceoryx_service_description(
   auto serviceDescriptionTuple = get_service_description_from_name_n_type(topic_name, type_name);
 
   return iox::capro::ServiceDescription(
-    iox::capro::IdString(iox::cxx::TruncateToCapacity, std::get<0>(serviceDescriptionTuple)),
-    iox::capro::IdString(iox::cxx::TruncateToCapacity, std::get<1>(serviceDescriptionTuple)),
-    iox::capro::IdString(iox::cxx::TruncateToCapacity, std::get<2>(serviceDescriptionTuple)));
+    iox::cxx::string<100>(iox::cxx::TruncateToCapacity, std::get<0>(serviceDescriptionTuple)),
```

Comment:
I think it would be better to use iox::capro::IdString_t. This is today the same but more future proof as this one is used in the ServiceDescription

Reply:
Agreened.